Contact.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+  <link rel="stylesheet" href="styles/main.css">
+</head>
+
+<body>
+  <div class="wrapper">
+
+    <header class="site-header">
+      <img src="images/download.jpeg" alt="logo-icon" width="77" height="77">
+      <nav class="site-navigation">
+        <ul>
+          <li><a href="Home.html">HOME</a></li>
+          <li><a href="About.html">ABOUT</a></li>
+          <li><a href="Contact.html">CONTACT</a></li>
+        </ul>
+      </nav>
```

Comment:
Nice work on the header, I think the main heading might be missing here, Is it part of the design?

Reply:
thanks for helpful comments, i ll fix it directly...